src/pages/debrief/cat-a-mod2/debrief.cat-a-mod2.page.ts

```diff
@@ -57,6 +57,7 @@ export class DebriefCatAMod2Page extends BasePageComponent {
   pageState: DebriefPageState;
   subscription: Subscription;
   isPassed: boolean;
+  category: TestCategory = TestCategory.EUAM2;
```

Comment:
why are we hard keying test category here? this doesn't see right and should already have access to it from pageState.category$